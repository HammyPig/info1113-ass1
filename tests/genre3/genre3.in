GENRE Fantasy
ADD COLLECTION books.csv
GENRE Fantasy
geNRE faNtasy
genre Historical Fiction