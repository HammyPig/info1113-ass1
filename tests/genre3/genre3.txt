recognises multi word genres, and can output multiple books per genre