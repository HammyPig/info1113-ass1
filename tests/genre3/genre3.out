user: No books in system.

user: 17 books successfully added.

user: The Hobbit (J.R.R. Tolkien)
Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: No books with genre faNtasy.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
